project_0a/project_0a_5.c

```diff
@@ -0,0 +1,31 @@
+/*
+Project 0a
+*/
+
+/*
+Exercise 5 - build a multiplication table for a given number. Use scanf() to
+read input from keyboard
+*/
+
+#include <stdio.h>
+
+int main() {
+  // Already answered this on question 2
```

Comment:
This note should be at the top of the file, it doesn't have anything to do with the main function.

If the same file solves two exercises, just name it as such instead of copying it. It bloats the review, and forces you to apply every fix twice, for no good reason.